core/templates/dev/head/pages/topics_and_skills_dashboard/SkillsListDirective.js

```diff
@@ -177,30 +175,10 @@ oppia.directive('skillsList', [
             modalInstance.result.then(function(result) {
               var skill = result.skill;
               var supersedingSkillId = result.supersedingSkillId;
-              var changeList = [{
-                cmd: 'update_skill_property',
-                property_name: 'superseding_skill_id',
-                old_value: '',
-                new_value: supersedingSkillId
-              }];
-              EditableSkillBackendApiService.updateSkill(
-                skill.id, skill.version,
-                'Added superseding skill id ' +
-                 supersedingSkillId + ' to skill.',
-                changeList
-              ).then(function() {
-                // Broadcast will update the skills so it will get the skill
```

Comment:
are we expecting to remove this comment?

Reply:
Done, added similar comments.